src/dev/src/app/forms/template-driven/template-driven.html

```diff
@@ -0,0 +1,26 @@
+<!--
+  ~ Copyright (c) 2016-2018 VMware, Inc. All Rights Reserved.
+  ~ This software is released under MIT license.
+  ~ The full license information can be found in LICENSE in the root directory of this project.
+  -->
+
+<h2>Text Input</h2>
+
+<form class="clr-form" #form="ngForm" (submit)="submit()">
```

Comment:
Just to be safe, can we test this for Reactive Forms too?

Reply:
Added new demos.